src/Components/AllReviews.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react";
+import { ReviewCard } from "./ReviewCard";
+
+export function AllReviews() {
+  const [reviews, setReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const reqURL = "https://board-game-reviews-brfi.onrender.com/api/reviews";
+
+  useEffect(() => {
+    getAllReviews().then(() => {
+      setIsLoading(false);
+    });
+  }, []);
+
+  const getAllReviews = async () => {
+    const response = await fetch(`${reqURL}`);

```

Comment:
It would be nice to have a seperate api file to hold any fetch functions in which can then just be called inside the components.